Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.7.3-slim
+
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV AIRFLOW_HOME=/src/airflow
+
+WORKDIR /src/app
```

Comment:
`/srv/app`?

Reply:
(there is a `/usr/src` in https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard but it's mostly kernel C source code)

Reply:
updated so that `/src/airflow` is the working directory.